federatedscope/core/worker/server.py

```diff
@@ -560,7 +560,7 @@ def eval(self):
                 # Preform evaluation in server
                 metrics = {}
                 for split in self._cfg.eval.split:
-                    eval_metrics = trainer.evaluate(
+                    eval_metrics = trainer.evaluate(mode=split,
```

Comment:
As we want to evaluate train split, I think the run_routine should not be `train`.